.emacs.d/lisp/user/user-config-system.el

```diff
@@ -8,6 +8,9 @@
   "~/.emacs.d/lisp/user/default-username"
   "The file that sets the default username for the machine.  (Ignored by git.)")
 
+(defvar kotct/directory "~/.emacs.d"
```

Comment:
The value and description don't seem to match here—is this specifically for the `.emacs.d` or the clone of `dot`? (Does it matter?)

Reply:
(Also, could this also get moved to `.emacs.d/lisp/package/git-update.el` as well?)

Reply:
uhm sure. imho we should have some higher `lisp/dot.el` in `lisp/` that contains all the "global stuff" including this and maybe `kotct/run-git` etc. this can be another ticket.

it's for the clone of `kotct/dot` or one of it's sub directories. we just need a directory within the git working directory. i think this is clear from the description and it's usage. 

done.